src/components/RollupProductDevDoc.js

```diff
@@ -0,0 +1,64 @@
+// Libraries
+import React from "react"
+import styled from "styled-components"
+
+// Components
+import Link from "./Link"
+
+// Data
+import optimisticRollupList from "../data/layer-2-data-files/optimistic-rollups.json"
+import zkRollupList from "../data/layer-2-data-files/zk-rollups.json"
+
+// Styles
+const H3 = styled.h3`
+  margin-top: 0;
+  margin-bottom: 1rem;
+`
+
+const ProductCard = styled.div`
+  margin: 1rem 0;
+  background: ${(props) => props.theme.colors.searchBackground};
+  display: flex;
+  flex-direction: row;
+`
+
+const Content = styled.div`
+  padding: 1rem;
+`
+
+const RollupProductDevDoc = ({ rollupType }) => {
+  const data = {
+    optimistic: optimisticRollupList,
+    zk: zkRollupList,
+  }
+
+  return (
+    <div>
+      {data[rollupType].map((rollup) => {
+        return (
+          <ProductCard>
+            <Content>
+              <div>
+                <H3>
+                  {rollup.name}
+                  {rollup.note.length > 0 && ` (${rollup.note})`}
+                </H3>
+                <div>
+                  <Link to={rollup.website}>Website</Link>
+                </div>
+                <div>
+                  <Link to={rollup.developerDocs}>Developer docs</Link>
+                </div>
+                <div>
+                  <Link to={rollup.l2beat}>L2BEAT</Link>
```

Comment:
```suggestion
                  <Link to={rollup.l2beat}>Technology and risk summary</Link>
```

I'd recommend a more descriptive link text vs. just "L2BEAT" which most people probably won't know what that means.

Reply:
I agree, will update with the new merged code, cant commit this suggestion after pulling in @minimalsm suggested changes 😅